lib/rubocop/cop/lint/duplicate_regexp_character_class_element.rb

```diff
@@ -32,26 +32,40 @@ def on_regexp(node)
           end
         end
 
+        # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
         def each_repeated_character_class_element_loc(node)
           node.parsed_tree&.each_expression do |expr|
-            next if expr.type != :set || expr.token == :intersection
+            next if skip_expression?(expr)
```

Comment:
Why factor this out if it's only called once?

Reply:
This seems to be called multiple times.